src/AutoComplete/AutoComplete.js

```diff
@@ -56,6 +56,10 @@ class AutoComplete extends Component {
      */
     dataSource: PropTypes.array.isRequired,
     /**
+     * Field map for search/results in objects list dataSource.
```

Comment:
I'm having trouble parsing that, could you perhaps reword it? Also, please could you document the expected object signature.